src/bin-utils/parser.js

```diff
@@ -117,6 +123,7 @@ function parse(rawArgv) {
       log.info(help(psConfig))
       return true
     }
+    const helpStyle = String(psConfig.options['help-style'] || 'all')
```

Comment:
Is this `|| 'all'` necessary considering the parser has `default: 'all'`? I'm pretty sure that `psConfig.options['help-style']` should always be defined.